database/migrations/2020_05_24_160133_add_gtid_to_users.php

```diff
@@ -14,7 +14,7 @@ class AddGtidToUsers extends Migration
     public function up(): void
     {
         Schema::table('users', static function (Blueprint $table): void {
-            $table->integer('gtid')->after('last_name');
+            $table->integer('gtid')->after('last_name')->nullable(true);
```

Comment:
This isn't actually shift-related, it just makes the migrations work on SQLite because I am lazy. If this is not desired I can remove it.

Reply:
This is fine with me as long as the UI requires it.